enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -0,0 +1,38 @@
+"""
+management command for assigning existing enterprise users to their respective enterprise roles.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from enterprise.constants import ENTERPRISE_ADMIN_ROLE, ENTERPRISE_DATA_API_ACCESS_GROUP, ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCustomerUser, SystemWideEnterpriseRole, SystemWideEnterpriseUserRoleAssignment
+
+
+class Command(BaseCommand):
+    """ management command class """
+    help = 'Assigns existing enterprise users to their specific roles.'
+
+    def handle(self, *args, **options):
+        try:
+            enterprise_admin_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_ADMIN_ROLE)
+            enterprise_learner_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_LEARNER_ROLE)
+        except SystemWideEnterpriseRole.DoesNotExist:
+            raise CommandError('The configured SystemWideEnterpriseRole role does not exist.')
+
+        # Assign admin role to users with enterprise data api access .
+        users_with_enterprise_access = User.objects.filter(groups__name=ENTERPRISE_DATA_API_ACCESS_GROUP)
```

Comment:
Open question: Should we also filter users for `enterprise_enrollment_api_access` group?

Reply:
@brittneyexline ^^

Reply:
for now no - the enrollment api access is more or less separate from how we "grant admin access" at this point.